src/components/AppPages.js

```diff
@@ -1,5 +1,6 @@
 export default {
     UPLOAD: '/new-job/files',
     SETTINGS: '/new-job/settings',
+    INSPECT: '/new-job/inspect',
```

Comment:
Let's update the url to `/jobs/:id/result-details` as described in requirements doc file (see route for result summary which is already in master)